networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,64 @@
+"""
+An edge cover of a graph is a set of edges such that every vertex of
+the graph is incident to at least one edge of the set.
+A minimum edge cover is an edge covering of smallest possible size.
+
+`Edge Cover <https://en.wikipedia.org/wiki/Edge_cover>`_
```

Comment:
Will `covering.py` only contain `minimum_edge_cover`?
If yes, then the filename should be `edge_covering.py`
If no, this text is redundant, should be put in the function docstring instead.